strawberry_django_plus/optimizer.py

```diff
@@ -55,6 +55,18 @@
 )
 from .utils.typing import TypeOrSequence
 
+try:
+    from django.contrib.contenttypes.fields import GenericForeignKey, GenericRelation
+except (ImportError, RuntimeError):
+    if not TYPE_CHECKING:
+        # stubs which will never match
+        class GenericForeignKey:
+            ...
+
+        class GenericRelation:
+            ...
```

Comment:
This is similar to what I commented below:
```suggestion
    GenericForeignKey = None
    GenericRelation = None
```

I think it is better to have splicity for type checkers that those were not imported, so the code can adapt accordingly.